includes/promoted-jobs/class-wp-job-manager-promoted-jobs-notifications.php

```diff
@@ -125,22 +124,18 @@ function( $key ) use ( $keys ) {
 	}
 
 	/**
-	 * Checks if we should send a notification to wpjobmanager.com after a post meta is updated.
+	 * Checks if we should send a notification to wpjobmanager.com after promoted meta key is deleted.
```

Comment:
Maybe it is worth documenting that the idea for this method is to detect when the job was unpromoted?

I initially found it confusing because I thought it would be cool to track whether `_promoted` changes from `0` to `1`, but..I'm not sure that's the goal of this method after reading these changes, right?

Reply:
Sure! WDYT? https://github.com/Automattic/WP-Job-Manager/pull/2562/commits/63db340d060cfcfe88bdbd69665306607c9bc9d7